frontend/src/diagram/sprotty/WebSocketDiagramServer.tsx

```diff
@@ -233,7 +233,11 @@ export class SiriusWebWebSocketDiagramServer extends ModelSource {
     const { element } = action;
     if (this.activeTool) {
       if (this.activeTool.__typename === 'CreateNodeTool') {
-        this.invokeTool(this.activeTool, element.id);
+        let diagramSourceElementId = null;
+        if (this.diagramSourceElement?.kind !== 'Diagram') {
+          diagramSourceElementId = this.diagramSourceElement.id;
+        }
+        this.invokeTool(this.activeTool, diagramSourceElementId);
```

Comment:
The behavior of the frontend should stay the same, this change is related to an internal API of the backend

Reply:
Done